### PR TITLE
Assign article category

### PR DESCRIPTION
**Pull request by DwinaTech**
I just change the branch because previous one has unrelated commits

### CODE REVIEW

File: server/models/Article.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+const CategoryModel = require('../models/Category');
 const { Schema } = mongoose;
 
 const schema = new Schema({
     title: String,
-    fullContent: String
+    fullContent: String,
+    categoryId: [{
+        text: String,
+        postedBy: {
```
Comment (kabaros) [inline server/models/Article.js]: why do you need postedBy?

Comment (kabaros) [reply server/models/Article.js]: and it's not an array either - the article belongs to one category so something like 
```js
categoryId: {
   type: ...
   ref: ....
}
```
should be enough

File: server/routes/admin.js [hunk 2]
```diff
@@ -35,9 +35,30 @@ router.post('/categories/add', function (req, res, next) {
     res.redirect("/");
     res.end()
   }
-  categoryClient.addCategory(query, callback);
+  categoryClint.addCategory(query, callback);
```
Comment (kabaros) [inline server/routes/admin.js]: I don't think this variable name is correct `categoryClint` - it won't work

File: server/routes/admin.js [hunk 3]
```diff
@@ -35,9 +35,30 @@ router.post('/categories/add', function (req, res, next) {
...
 });
 
+/*===========================
+        Add article
+===========================*/
+router.get('/articles/add', (req, res, next) => {
```
Comment (kabaros) [inline server/routes/admin.js]: you have two routes for handling /articles/add now - the new one, and the one below (line 62), which one is correct? remove the other one

**Review by kabaros — CHANGES_REQUESTED** [review]
few more changes - also make sure you test it and see the output in the DB

**Comment by DwinaTech** [comment]
[ADMIN / Assign Article to a Category](https://trello.com/c/DUqbTmDg/24-admin-assign-article-to-a-category)

**Review by kabaros — APPROVED** [review]

**Pull request merged**